pages/how_do_mentor_queues_work.md

```diff
@@ -3,5 +3,5 @@ Normally, though, they work through either a track's queue (e.g. all the Ruby su
 
 Queues are ordered algorithmically, but fundamentally acts like first-in-first-out queues with core exercises prioritized over side exercises, and those submitted in "Mentored Mode" prioritized over those in "Practice Mode".
 
-Because there is no strict linear queue, and because mentors' rates of mentoring and mentoring behavior changes daily, we currently do not provide time estimates.
+Because there is no strict linear queue, and because mentors' rates of mentoring and mentoring behavior changes daily, we currently do not always provide time estimates.
```

Comment:
```suggestion
We provide time estimates for Core Exercises based on the median response time over recent weeks, and these can sometimes be inconsistent with the time you may wait. Because there is no strict linear queue, and because mentors' rates of mentoring and mentoring behavior changes daily, these estimates should be treated as our "best guess", rather than any guarantee.